py_feels/binary/time_series.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jan 20 14:50:21 2018
+
+Pulls feels and tweet times from the bit_feels database and performs
+a time-window average of the sentiments for plotting. pushes means,
+standard devaitions, classifier names and bin times into the 'stats'
+table of bit_feels. Takes two arguments: window and env
+
+window is the width of the time-bin in hours,
+env is "dev" or "prod"
+
+@author: rcehemann
+"""
+
+import pandas as pd
+from numpy import linspace, digitize, mean, std
+from datetime import datetime
+from sys import argv
+from sqlalchemy import create_engine
+
+# check environment, parse arguments
+if len(argv) < 2:
+    window = 48.
+    env = "bit_feels_dev"
+elif len(argv) < 3 or argv[2] == "dev":
+    window = argv[1]
+    env = "bit_feels_dev"
+elif argv[2] == "prod":
+    window = argv[1]
+    env = "bit_feels"
+else:
+    raise EnvironmentError("second argument should be 'dev' or 'prod'")
+
+# set up SQL connection
+engine = create_engine('postgresql+psycopg2://wojak:@localhost/' + env)
+
+def ts_to_td(inserted_at):
+    """
+        returns the total elapsed seconds from
+        the timestamp to present time.
+    """
+    return (inserted_at - datetime.now()).total_seconds()
+
+def time_window(times, window=24.):
```

Comment:
Isn't this setting the default window to 24?

Reply:
only for the function, but there's no reason to have window be a kwarg here. will change